hub/server.go

```diff
@@ -58,7 +58,7 @@ func (h *Hub) Serve() {
 			h.server.TLSConfig = certManager.TLSConfig()
 
 			// Mandatory for Let's Encrypt http-01 challenge
-			go http.ListenAndServe(":http", certManager.HTTPHandler(nil))
+			go http.ListenAndServe(h.options.AcmeAddr, certManager.HTTPHandler(nil))
```

Comment:
I'm not sure it is worth it, by the spec the acme HTTP challenge can only be resolved on the `80` port. At least, we should find a better option name, such as `AcmeHTTP01Addr`.

Reply:
This change had 2 purposes: 
- being able to test that part (we can't start a listener on port 80 without running test-suite with root user)
- allows several hubs running on the same server. Off course, a loadbalancer/proxy on top of the server should  route the traffic to the right port

Reply:
you may prefer a dedicated PR?

Reply:
moved in #192